.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,30 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: Bug Report
+about: Create a report to help us improve Semester.ly!
 title: ''
-labels: ''
+labels: 'bug'
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**Describe the Bug**
+
```

Comment:
Might want to keep this for consistency sake if every other section has helper text, but I think this one's up to you

Reply:
Alright added it all back but added an extra newline to give them some extra room to view each section separately.